src/components/Editor.js

```diff
@@ -33,6 +39,7 @@ export default connect(
   state => {
     return {
       isLoadingRuntime: state.loadApi.stage === 'started',
+      isLoadedRuntime: state.loadApi.stage === 'finished',
```

Comment:
can you stick these 'finished' and 'started' strings into a constant somewhere? Probably in the reducer file.